### ISSUE TITLE
The  scale factor for KITTI Evaluation Server Benchmark

### ISSUE THREAD

**Issue by kamiLight** [nianticlabs/monodepth2#376]
hi, in the section D.3. KITTI Evaluation Server Benchmark, you mention "Because the evaluation server does not do median scaling(required for monocular methods), we needed a way to findthe correct scaling for our mono-only model, which makes unscaled predictions. We make predictions with our mono-model on 1,000 images from the KITTI training set which have ground truth depths available, and for each of the 1,000images we find the scale factor which best align the depthmaps [76]. Finally, we take the median of these 1,000 scale factors as the single factor which we use to scale all pre-dictions from our mono model. "

but in evaluate_depth.py,
```
elif opt.eval_split == 'benchmark':
        save_dir = os.path.join(opt.load_weights_folder, "benchmark_predictions")
        print("-> Saving out benchmark predictions to {}".format(save_dir))
        if not os.path.exists(save_dir):
            os.makedirs(save_dir)

        for idx in range(len(pred_disps)):
            disp_resized = cv2.resize(pred_disps[idx], (1216, 352))
            depth = STEREO_SCALE_FACTOR / disp_resized
            depth = np.clip(depth, 0, 80)
            depth = np.uint16(depth * 25)
```

when using the benchmark splits to evaluation, the scale factor is the STEREO_SCALE_FACTOR which is 5.4, it doesn't seem like the scale factor computed by the 1,000 images.
So could you release the correct scale factor for mono-only model or the code computing this value?

**Comment by mrharicot** [comment]
Hi,

I am afraid I don't have this scale factor handy, however you could reproduce the way we computed it by simply running the training dataloader for 1000 images (with shuffling enabled) and computing the median of the median scaling.

I hope this helps.

**Comment by kamiLight** [comment]
Thanks for your response, i will reproduce this evaluation according to your methods.
